### PR TITLE
[PL-48751] DEL 82700 Release Notes

### PR DESCRIPTION
**Pull request by saifeemustafaqharness**
[PL-48751](https://harness.atlassian.net/browse/PL-48751)

https://harness.atlassian.net/wiki/spaces/PD/pages/21724594239/NG+Release+SAAS+April+1st+2024+for+1.32.x-+Build#NG-Delegate-dependent-changes--

For reviewers, a preview is available:
https://661eba4d73bf915668b431e7--harness-developer.netlify.app/release-notes/delegate#harness-version-1335-harness-delegate-version-240482707-

## What Type of PR is This?

- [ ] Issue
- [ ] Feature
- [x] Maintenance/Chore

If tied to an Issue, list the Issue(s) here:

* *Issue(s)*

## House Keeping
Some items to keep track of. Screen shots of changes are optional but would help the maintainers review quicker. 

- [x] Tested Locally
- [ ] *Optional* Screenshot. 

[PL-48751]: https://harness.atlassian.net/browse/PL-48751?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: release-notes/delegate.md [hunk 1]
```diff
@@ -70,6 +71,16 @@ import Deleos from '/docs/platform/shared/delegate-legacy-eos.md'
 
 ## April 2024
 
+### Harness version 1.33.x, Harness Delegate version 24.03.82700 <!--  April 09, 2024 -->
+
+#### Fixed issues
+
+- Slack channel notifications failed due to an error related to explicitly setting the "Host" header as "hooks.slack.com". We have removed the explicit "Host" header setting to support both Slack-specific webhook URLs and regular URLs, resolving the issue in this version. (PL-47914)
```
Comment (brian-f-harness) [inline release-notes/delegate.md:78]:
```suggestion
- Slack channel notifications failed due to an error related to explicitly setting the Host header as `hooks.slack.com`. We have removed the explicit Host header setting to support both Slack-specific webhook URLs and regular URLs, resolving the issue. (PL-47914)
```

**Comment by saifeemustafaqharness** [comment]
Can we proceed to merge this? @pratmit @brian-f-harness

**Comment by brian-f-harness** [comment]
> Can we proceed to merge this? @pratmit @brian-f-harness

@saifeemustafaqharness no, the 1.33.x PL release was rolled back.

https://harness.slack.com/archives/C04R4UDE1FT/p1712811171196209

File: release-notes/delegate.md [hunk 2]
```diff
@@ -70,6 +71,16 @@ import Deleos from '/docs/platform/shared/delegate-legacy-eos.md'
 
 ## April 2024
 
+### Harness version 1.33.x, Harness Delegate version 24.03.82700 <!--  April 09, 2024 -->
```
Comment (brian-f-harness) [inline release-notes/delegate.md:74]:
```suggestion
### Harness version 1.33.x, Harness Delegate version 24.04.82703 <!--  April xx, 2024 -->
```

File: release-notes/delegate.md [hunk 3]
```diff
@@ -1,7 +1,8 @@
 ---
 title: Delegate release notes
 sidebar_label: Delegate
-date: 2024-04-04T10:00
+tags: [NextGen, "Delegate"]
+date: 2024-04-09T10:00
```
Comment (brian-f-harness) [inline release-notes/delegate.md:5]:
```suggestion
date: 2024-04-xxT10:00
```

**Review by brian-f-harness — COMMENTED** [review]
will need to update date in metadata and comments on release day
also need to update delegate verno

**Comment by saifeemustafaqharness** [comment]
> > Can we proceed to merge this? @pratmit @brian-f-harness
> 
> @saifeemustafaqharness no, the 1.33.x PL release was rolled back.
> 
> https://harness.slack.com/archives/C04R4UDE1FT/p1712811171196209

But they did say that Delegate release won't be affected and it will be released as scheduled: https://harness.slack.com/archives/C011SD1S1U3/p1712653583667579?thread_ts=1712567638.968429&cid=C011SD1S1U3

**Comment by brian-f-harness** [comment]
> > > Can we proceed to merge this? @pratmit @brian-f-harness
> > 
> > 
> > @saifeemustafaqharness no, the 1.33.x PL release was rolled back.
> > https://harness.slack.com/archives/C04R4UDE1FT/p1712811171196209
> 
> But they did say that Delegate release won't be affected and it will be released as scheduled: https://harness.slack.com/archives/C011SD1S1U3/p1712653583667579?thread_ts=1712567638.968429&cid=C011SD1S1U3

That conversation was 2 days ago. Delegate hasn't been released: https://hub.docker.com/r/harness/delegate/tags. 1.33.x was deployed and rolled back today. When 1.33.x is deployed, we can publish approved delegate and PL RN.

**Comment by brian-f-harness** [comment]
@saifeemustafaqharness Suggest following up on this item before review:
https://harness.atlassian.net/browse/PL-48935?focusedCommentId=710519 - check with Marko for RN requirements

File: release-notes/delegate.md [hunk 4]
```diff
@@ -70,6 +71,16 @@ import Deleos from '/docs/platform/shared/delegate-legacy-eos.md'
 
 ## April 2024
 
+### Harness version 1.33.x, Harness Delegate version 24.04.82703 <!--  April 15, 2024 -->
```
Comment (brian-f-harness) [inline release-notes/delegate.md:74]:
```suggestion
### Harness version 1.33.5, Harness Delegate version 24.04.82703 <!--  April 15, 2024 -->
```

Comment (brian-f-harness) [reply release-notes/delegate.md:74]: 24.04.82707 is the delegate version in QA. Verify the release version with @bharatgoelharness.

Comment (brian-f-harness) [reply release-notes/delegate.md:74]: Please add https://harness.atlassian.net/browse/PL-48781

File: release-notes/delegate.md [hunk 5]
```diff
@@ -1,7 +1,8 @@
 ---
 title: Delegate release notes
 sidebar_label: Delegate
-date: 2024-04-12T10:00
+tags: [NextGen, "Delegate"]
+date: 2024-04-15T10:00
```
Comment (brian-f-harness) [inline release-notes/delegate.md:5]:
```suggestion
date: 2024-04-16T10:00
```

File: release-notes/delegate.md [hunk 6]
```diff
@@ -70,6 +71,18 @@ import Deleos from '/docs/platform/shared/delegate-legacy-eos.md'
...
 
+### Harness version 1.33.5, Harness Delegate version 24.04.82707 <!--  April 16, 2024 -->
+
+#### Fixed issues
+
+- Slack channel notifications failed due to an error related to explicitly setting the Host header as `hooks.slack.com`. We have removed the explicit Host header setting to support both Slack-specific webhook URLs and regular URLs, resolving the issue. (PL-47914)
+
+- In SCIM, creating a new user with special characters in their name failed, preventing the user from being added to Harness and resulting in discrepancies in user group membership between the Identity Provider and Harness. The name of a user will be sanitized if it does not follow Harness naming conventions during user addition flows. (PL-47614)
+
+- Builds triggered by Bitbucket Server push events have correct date information in the build history. This issue occurred due to missing date information in the `commits` object returned by the Bitbucket Server API. (CI-11556, ZD-58798)
```
Comment (brian-f-harness) [inline release-notes/delegate.md:82]:
```suggestion
- Builds triggered by Bitbucket Server push events had incorrect date information in the build history. This issue occurred due to missing date information in the `commits` object returned by the Bitbucket Server API. (CI-11556, ZD-58798)
```

File: release-notes/delegate.md (hunk 5 above)
Comment (brian-f-harness) [reply release-notes/delegate.md:5]: @saifeemustafaqharness Please update the date in the metadata. This controls the RSS feed. Thanks.

**Review by brian-f-harness — APPROVED** [review]

**Review by pratmit — APPROVED** [review]
LGTM

**Pull request merged**